controller/tests/array_action/svc/array_mediator_svc_test.py

```diff
@@ -732,14 +767,18 @@ def _get_detailed_hosts_list_result(self, hosts_dict):
             current_element = SVCListResultsElement()
             current_element.add(HOST_ID_PARAM, host_dict.get(HOST_ID_PARAM))
             current_element.add(HOST_NAME_PARAM, host_dict.get(HOST_NAME_PARAM))
-            iscsi_names_list = host_dict.get(HOST_ISCSI_NAMES_PARAM)
-            if iscsi_names_list:
-                for iscsi_name in iscsi_names_list:
-                    current_element.add(HOST_ISCSI_NAMES_PARAM, iscsi_name)
+            nvme_nqn_list = host_dict.get(HOST_NQN_PARAM)
+            if nvme_nqn_list:
+                for nqn in nvme_nqn_list:
+                    current_element.add(HOST_NQN_PARAM, nqn)
```

Comment:
not a must (it's a gap in existing code), but I think this can be better:
```suggestion
            nvme_nqn_list = host_dict.get(HOST_NQN_PARAM, [])
            for nqn in nvme_nqn_list:
                current_element.add(HOST_NQN_PARAM, nqn)
```

Reply:
nice! done